.github/workflows/ci.yml

```diff
@@ -46,18 +52,18 @@ jobs:
           npm ci
           npm run lint
 
-
   drupal_test:
     runs-on: ubuntu-latest
+    needs: [drupal_codequality]
```

Comment:
I had removed this in the past because code quality rarely fails in practice (local checks catch errors upfront). So, waiting for this check to be done before beginning `drupal_test` was a little annoying.

Reply:
Ok understood. will remove it.